packages/core/src/app/order/OrderConfirmation.tsx

```diff
@@ -143,11 +143,11 @@ class OrderConfirmation extends Component<
                         }) }
 
                         <div className="continueButtonContainer">
-                            <a href={ siteLink } target="_top">
-                                <Button variant={ ButtonVariant.Secondary }>
+                            <form action={ siteLink } method="get" target="_top">
+                                <Button type="submit" variant={ ButtonVariant.Secondary }>
                                     <TranslatedString id="order_confirmation.continue_shopping" />
                                 </Button>
-                            </a>
+                            </form>
```

Comment:
Changing `<button>` to `<a>` will make its text colour to be blue, even with the same class names. Hence I went for the `form` solution.

Reply:
Again what was the issue with `a` before?

Reply:
According to the audit doc, A `<Button>` should not not be in `a`. 
> "HTML does not allow an interactive element to be nested in a link. This will cause significant issues for some assistive technology.

> Remove the button markup and keep the link and the text. "